src/Pages/EventMain/services/getFilteredEvents.js

```diff
@@ -1,8 +1,17 @@
 export function getFilteredEvents(
   eventArray,
   eventFilter,
+  eventSearch,
+  multipleEventSearch,
   categoryfilteredEvents
 ) {
+  if (eventSearch && eventSearch.length > 0)
```

Comment:
Do you need the logical and here? I think if "eventSearch.length >0",  "eventSearch" will automatically be true.

Reply:
yes, it'S needed here